auditlog_ext_nrq/tests/test_auditlog_ext_nrq.py

```diff
@@ -64,3 +68,51 @@ def test_03_unlink_record(self):
             ('res_id', '=', self.translation_record.id),
         ], limit=1, order='id desc')
         self.assertEqual(log.log_category, 'unlink')
+
+    def test_04_create_vals(self):
+        """
+            Test old value text,new value text with datetime field data
+            Test old value text,new value text without datetime field data
+        """
+        tz = 'Japan'
+        self.test_user = self.env.user.copy()
+        self.env.user.write({'tz': tz})
```

Comment:
Why don't we set the timezone to the superuser, following the expected usage?

Reply:
Yes, the default timezone for the super user is Europe/Brussels so change to Japan.